src/openrct2/scripting/Duktape.hpp

```diff
@@ -204,6 +205,52 @@ namespace OpenRCT2::Scripting
         }
     };
 
+    const static EnumMap<RatingsModifierType> RatingsModifierTypeToName{
```

Comment:
I'd make the enums in the same style of [other enums](https://github.com/OpenRCT2/OpenRCT2/blob/3b6b6e080f6802acf662ad8f114c5cd18d19c552/distribution/openrct2.d.ts#L589), e.g. `snake_case`.